src/Controller/Services.php

```diff
@@ -73,6 +73,14 @@ public function processRequest(Request $request, RouteMatchInterface $route_matc
     }
     // Get the data from the plugin.
     $data = $service_def->processRequest($request, $route_match, $this->serializer);
+    $code = $service_def->getPluginDefinition()['response_code'];
```

Comment:
It's the response_code from the plugin definition. i.e. 200 unless a specific plugin overrides it.